src/joint.dia.link.js

```diff
@@ -1446,7 +1436,7 @@ joint.dia.LinkView = joint.dia.CellView.extend({
         // if are simulating pointerdown on a link during a magnet click, skip link interactions
         if (evt.target.getAttribute('magnet') != null) return;
 
-        var interactive = _.isFunction(this.options.interactive) ? this.options.interactive(this, 'pointerdown') : this.options.interactive;
+        var interactive = _.isFunction(this.options.interactive) ? this.options.interactive(this) : this.options.interactive;
```

Comment:
Okie dokie. So just remove the `var = interactive ..` line? Don't add any new `can()` instances there?